### DISCUSSION TITLE
ET-Open updates

### DISCUSSION THREAD

**Discussion by gobrien1231** [Security-Onion-Solutions/securityonion#6645] [Unsupported Versions] [answered]
Hey all, super basic question. Does SO automatically retrieve updated suricata rules when configured for ET-Open?

**Comment by dougburks** [comment, marked as answer]
Yes, I've updated https://docs.securityonion.net/en/2.3/rules.html

**Reply by gobrien1231** [reply]
Thanks Doug!
